dev-portal/scripts/deploy-stack.js

```diff
@@ -0,0 +1,33 @@
+const { execute, r } = require('./utils.js')
+
+const deployerConfig = require('../deployer.config.js')
+const writeConfig = require('./write-config.js')
+
+const samTemplate = deployerConfig.samTemplate || r(`../../cloudformation/template.yaml`)
+const packageConfig = deployerConfig.packageConfig || r(`../../cloudformation/packaged.yaml`)
+const stackName = deployerConfig.stackName
+const buildAssetsBucket = deployerConfig.buildAssetsBucket
+const siteAssetsBucket = deployerConfig.siteAssetsBucket
```

Comment:
We should really standardize on names for these...

Reply:
Yeah... `1st, 2nd and 3rd` don't really cut it, do they? The pattern I used here is differentiating by purpose. 1st - build (b/c it has the pre-deployment assets); 2nd - site (b/c it hosts the site); 3rd - api (b/c it has the catalog and potentiall? the api-related info). Not attached.